repo/packages/K/kafka/0/marathon.json

```diff
@@ -3,18 +3,18 @@
   "cpus": 0.5,
   "mem": 256,
   "instances": 1,
-  "cmd": "jre/bin/java -jar kafka-mesos*.jar scheduler --master={{kafka/master}} --zk={{kafka/zk}} --api={{kafka/api}} --storage={{kafka/storage}} --jre={{kafka/jre-file}} {{kafka/otherOptions}}",
-  "uris": ["{{kafka/jre}}", "{{kafka/kafka}}", "{{kafka/jar}}"],
+  "cmd": "jre/bin/java -jar kafka-mesos*.jar scheduler --master={{mesos.master}} --zk={{kafka.zk}} --api={{kafka.api}} --storage=\"{{kafka.storage}}\" --jre={{kafka.jre-file}} {{#kafka.user}}--user=\"{{kafka.user}}\"{{/kafka.user}} --framework-name=\"{{kafka.framework-name}}\" --framework-role=\"{{kafka.framework-role}}\" {{#kafka.principal}}--principal=\"{{kafka.principal}}\"{{/kafka.principal}} {{#kafka.secret}}--secret=\"{{kafka.secret}}\"{{/kafka.secret}} {{kafka.other-options}}",
+  "uris": ["{{kafka.jre}}", "{{kafka.kafka}}", "{{kafka.jar}}"],
   "ports": [0],
   "healthChecks": [
     {
       "protocol": "HTTP",
-      "path": "/api/brokers/status",
+      "path": "/health",
       "gracePeriodSeconds": 120,
       "intervalSeconds": 60,
       "portIndex": 0,
       "timeoutSeconds": 30,
-      "maxConsecutiveFailures": 3
+      "maxConsecutiveFailures": 0
     }
```

Comment:
Both health checks can be listed here with the correct parameters so that when the cluster is unhealthy marathon doesn't try and restart the scheduler, but when the scheduler is unhealthy marathon can restart it.